### PR TITLE
fix message sending from message review

### LINKED ISSUE #1337: Fix message sending from message review
As part of RFC #1320 and the project [Bring work in WorkingFamilies/Spoke fork into MoveOnOrg/Spoke](https://github.com/MoveOnOrg/Spoke/projects/14), merge the work in this WorkingFamiles/Spoke PR:
- [125](https://github.com/WorkingFamilies/Spoke/pull/125)

### PR DESCRIPTION
**Pull request by lperson**
# Fixes #1337 

## Description

Message sending from message review wasn't working work because objects passed into MessageResponse did not have the expected fields.

# Checklist:

- [x] I have manually tested my changes on desktop and mobile
- [x] The test suite passes locally with my changes
- [ ] Not a lot to see here.  No functional change other than before you'd see an error, now you don't.  ~If my change is a UI change, I have attached a screenshot to the description section of this pull request~
- [x] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
- [N/A] ~I have made any necessary changes to the documentation~
- [ ] Although for the record I believe tests are always necessary, this is a small, isolated fix so I didn't write a test. ~I have added tests that prove my fix is effective or that my feature works~
- [N/A] ~My PR is labeled [WIP] if it is in progress~

### CODE REVIEW

**Comment by azuzunaga** [comment]
Reviewing...

File: src/containers/AssignmentTexterContact.jsx [hunk 1]
```diff
@@ -487,9 +488,14 @@ export class AssignmentTexterContact extends React.Component {
...
       await this.handleSubmitSurveys();
-      await this.props.mutations.createOptOut(optOut, contact.id);
+      await this.props.mutations.createOptOut(
+        this.props.organizationId,
+        optOut,
+        contact.id
+      );
       this.props.onFinishContact(contact.id);
     } catch (e) {
+      console.log("ERRROR", e);
```
Comment (azuzunaga) [inline src/containers/AssignmentTexterContact.jsx:498]:
```suggestion
      console.log("ERROR", e);
```

File: src/containers/IncomingMessageList/MessageResponse.jsx [hunk 2]
```diff
@@ -32,20 +32,20 @@ class MessageResponse extends Component {
   }
 
   createMessageToContact(text) {
-    const { contact, texter } = this.props.conversation;
+    const contact = this.props.conversation;
```
Comment (azuzunaga) [inline src/containers/IncomingMessageList/MessageResponse.jsx:35]:
```suggestion
    const { contact: { texter: { id: userId } } } = this.props.conversation;
```

File: src/containers/IncomingMessageList/MessageResponse.jsx [hunk 3]
```diff
@@ -32,20 +32,20 @@ class MessageResponse extends Component {
...
 
     return {
       assignmentId: contact.assignmentId,
       contactNumber: contact.cell,
-      userId: texter.id,
+      userId: contact.texter.id,
```
Comment (azuzunaga) [inline src/containers/IncomingMessageList/MessageResponse.jsx:40]:
```suggestion
      userId
```

**Review by azuzunaga — APPROVED** [review]
Looks good. Just some minor questions and suggestions to match code style between functions.

**Comment by lperson** [comment]
Closing in favor of #1412

**Pull request closed without merging**